provision-contest/ansible/roles/domserver/templates/nginx-domjudge.conf.j2

```diff
@@ -7,6 +7,36 @@ upstream domjudge {
 	server unix:/var/run/php-fpm-domjudge.sock; # if using with etc/domjudge-fpm.conf
 }
 
+{% if DOMSERVER_LOADBALANCING %}
+upstream domjudge-loadbalanced {
+	least_conn;
+{% for host in groups['domserver'] %}
+	server {{ hostvars[host].ansible_host }}:81;
+{% endfor %}
+}
+
+server {
+	listen   81;
```

Comment:
We would now strip the SSL and send the requests plaintext through the network right?

Reply:
We would. Is that a problem? Teams can’t access this part of the network anyway. Doing double ssl is not something I like

Reply:
Also changed this to SSL.